packages/frontpage/lib/schema.ts

```diff
@@ -267,3 +265,12 @@ export const Report = sqliteTable("reports", {
     enum: ["pending", "accepted", "rejected"],
   }).default("pending"),
 });
+
+export const Notification = sqliteTable("notifications", {
+  id: integer("id").primaryKey(),
+  did: did("did").notNull(),
+  createdAt: dateIsoText("created_at").notNull().default(nowAsIsoString),
+  readAt: dateIsoText("read_at"),
```

Comment:
you can default this to strftime and leave it nullable maybe?

Reply:
Can't default it because it needs to be null to indicate not read.